src/mem/Pool.h

```diff
@@ -166,9 +166,14 @@ class MemPools
     void setDefaultPoolChunking(bool const &);
 
     MemImplementingAllocator *pools = nullptr;
-    ssize_t mem_idle_limit = (2 << 20) /* 2MB */;
     int poolCount = 0;
     bool defaultIsChunked = false;
+
+private:
+    /// Limit for allocated (but unused) memory in memory pools.
+    /// Initial value is 2MB until first configuration,
+    /// \see squid.conf memory_pools_limit directive.
+    ssize_t poolIdleLimit = (2 << 20);
```

Comment:
Please avoid fooX inside Foo:

```suggestion
    ssize_t idleLimit_ = (2 << 20);
```